nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py

```diff
@@ -190,6 +204,8 @@ def get_ipa_punctuation_list(locale):
     elif locale == "es-ES":
         # ref: https://en.wikipedia.org/wiki/Spanish_orthography#Punctuation
         punct_set.update(['¿', '¡'])
+    elif locale == "fr-FR":
+        punct_set.update(['–', '“', '”', '…', '̀', '́', '̂', '̈', '̧'])
```

Comment:
Regarding the extra punctuations used in French, could you pls add comments of unicode for each punctuation just as what "de-DE" did? It is not easy to pinpoint the difference between similar surfaces, such as below. Thanks!       
```
'‒',  # figure dash, U+2012, decimal 8210
'–',  # en dash, U+2013, decimal 8211
```